src/earn/EarnActivePool.tsx

```diff
@@ -66,19 +76,19 @@ export default function EarnActivePool() {
               <TokenDisplay
                 amount={poolToken.balance}
                 showLocalAmount={false}
-                testID={`${networkConfig.arbUsdcTokenId}-totalAmount`}
-                tokenId={networkConfig.arbUsdcTokenId}
+                testID={`${depositTokenId}-totalAmount`}
+                tokenId={depositTokenId}
               />
             )}
           </View>
           <View style={styles.row}>
             {asyncPoolInfo.error && <View />}
             {asyncPoolInfo.loading && <PoolDetailsLoading />}
-            {asyncPoolInfo.result && (
+            {asyncPoolInfo.result && asyncPoolInfo.result.apy && (
```

Comment:
this will crash if apy is 0

Reply:
I thought the same, but in testing the .toFixed(2) will cause 0 to display as '0.00'